requirements.txt

```diff
@@ -3,4 +3,5 @@ ovos-utils~=0.0, >=0.0.35
 ovos_workshop~=0.0, >=0.0.11
 ovos-classifiers
 ovos-translate-server-plugin
-ovos-config>=0.0.11a10
\ No newline at end of file
+ovos-config>=0.0.11a10
+neon-solver-ddg-plugin>=0.0.2a1
```

Comment:
I think we don't want to have the Neon solver as a dependency. But @JarbasAl will confirm.

Reply:
wouldn't load without it installed, but may be missing something?